src/explore-education-statistics-frontend/src/components/PrintThisPageAnalytics.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+
+const PrintThisPage = () => {
+  return (
+    <div className={classNames('govuk-!-margin-top-6', 'dfe-print-hidden')}>
+      <a
+        href="#"
+        onClick={() => {
+          window.print();
```

Comment:
I would log the event right _before_ calling print, just incase the user closes the tab while the print window is open and the log event may never be called